code/src/index.css

```diff
@@ -1,13 +1,210 @@
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+	background-color: black;
+	margin: 0;
+	font-family: 'Montserrat', sans-serif;
+	/* font-family: -apple-system, BlinkMacSystemFont, "Montserrat", "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    sans-serif; */
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	box-sizing: border-box;
+}
+
+/* code {
+	font-family: source-code-pro, Menlo, Monaco, Consolas, 'Montserrat',
+		'Courier New', monospace;
+} */
+
+.poster-img {
+	width: 100%;
+	height: 100%;
+	display: inherit;
+	object-fit: cover;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.movie-list-container {
+	margin: 0 auto;
+}
+
+.movie {
+	position: relative;
+	display: inherit;
+}
+
+.overlay-text {
+	color: white;
+	padding: 2rem;
+	transition: 0.3s ease;
+	font-size: 1.2em;
+}
+
+.overlay-text p {
+	font-size: 1rem;
+}
+
+.detail-poster {
+	width: 90%;
+	max-width: 25rem;
+	box-shadow: 0rem 0rem 2rem 1rem rgba(0, 0, 0, 0.521);
+}
+
+.detail-text {
+	font-size: 1.15rem;
+	line-height: 1.6;
+	max-width: 55em;
+	width: 90%;
+}
+
+.backdrop {
+	z-index: -1;
+	width: 100%;
+	height: 100%;
+}
+
+.movie-info-container {
+	display: grid;
+	min-height: 90vh;
+	padding: 2rem 1rem;
+}
+
+.movie-info {
+	padding-top: 3rem;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	color: white;
+	gap: 2rem;
+}
+
+.overlay {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	/* width: 100%; */
+	background: linear-gradient(
+		180deg,
+		rgba(0, 0, 0, 0) 0%,
+		rgba(0, 0, 0, 1) 100%
+	);
+	transition: 0.3s ease;
+	display: flex;
+	align-items: flex-end;
+}
+
+.rating {
+	/* border: 1px solid white;
+	border-radius: 2rem; */
+	font-size: 1.5rem;
+	font-style: italic;
+	padding: 0 1rem;
+	font-weight: 400;
+}
+
+.detail-text h2 {
+	margin: 0;
+	font-size: 2rem;
+}
+
+.error {
+	font-size: 2rem;
+	color: white;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	min-height: 100vh;
+	flex-direction: column;
+}
+
+.loading-container {
+	display: grid;
+	justify-content: center;
+	align-items: center;
+	min-height: 100vh;
+}
+
+.loading {
+	border: 20px solid #f3f3f3;
+	border-top: 20px solid #555;
+	border-radius: 50%;
+	width: 80px;
+	height: 80px;
+	animation: spin 1.5s linear infinite;
+}
+
+@keyframes spin {
+	0% {
+		transform: rotate(0deg);
+	}
+	100% {
+		transform: rotate(360deg);
+	}
+}
+
+@media (min-width: 35em) {
+	.movie-list-container {
+		max-width: 100%;
+		display: grid;
+		grid-template-columns: 1fr 1fr;
+	}
+}
+
+@media (min-width: 55em) {
+	.movie-info {
+		flex-direction: row;
+		align-items: end;
+	}
+
+	.movie-info-container {
+		padding: 2rem;
+	}
+}
+
+@media (min-width: 75em) {
```

Comment:
EM for media queries! I've never seen that before. WHOA! Somebody's gonna have to explain this to me. No, I'm serious - please reply to this comment at some point and explain ems instead of px for media query. Love it!